config/manifests/bases/codeflare-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: codeflare-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: InstaScale is the Schema for the instascales API
+      displayName: Insta Scale
```

Comment:
Is there a reason for the space here? (InstaScale)

Reply:
Ah, the reason would be that the underlying display name in the CRD is wrong. (this was auto generated from those).

Reply:
My vote would be a follow up PR later to fix them up

Reply:
Opened #59